### ISSUE TITLE
Toeci

### ISSUE THREAD

**Issue by rainit2006** [rainit2006/Anything#6]

**Comment by rainit2006** [comment]
How much is the lawyer charging us?
I already got a verbal okay from John Sutton.




in the heat of the city
xxxx tend to be like this. 
should be any minute now.
make ends meet
生活の収支を合わせる、収入の範囲内でやりくりする［やっていく］

It's good to run into you
not until later on
I already got a verbal okay from John Sutton. 
here on the premises
Oh? I wasn't expecting them until three.
 How much is the lawyer charging us?
It takes an hour to charge the battery. 
I just wanted to make sure you ...
and not one to be taken lightly.
Do you do that sort of thing?
Well, for instance
I’ll go get it for you.
 so what would that do?
I also ordered the catering package
Yes, that’s all taken care of.
What’s in the big envelope?
I think he is.
A consulting firm in Boston. 
I think it begins at 6
They’re going fast
Are tickets for the evening show sold out yet? 
What’s in the big envelope? 
employee orientation
Rent is due every month on the 15th unless the 15th falls on a weekend. 
Who is the intended audience for the talk? 
it will be better able to compete against 
 If you would
The man is gesturing as he speaks.
The man is posing for a picture. 
The man is taping the scenery.
The man is standing in a train station.
They are in the middle of a discussion. 
They are applauding the orchestra. 
They are standing next to each other. 
They are watching a parade on the street.
The woman is doing two things at once. 
The woman is wearing a suit and glasses. 
Do you think you could plug in the projector for me? 
Sure, where's an outlet?
Excite Travel is offering a four-day cruise to Puerto Rico and the Virgin Islands.
When should we start the slide show
Would you rather look for a jacket at the mall or order one online?
I'll look at the mall first.
 She wanted only a little.
I might have some, what size are they?
Why are we getting paid early?
I need some batteries for my camera.
The show is on Monday.
I always visit her.
We haven't gotten to that part yet. 
Who's going to show the new employees around? 谁会带新员工熟悉环境呢？
The test results won't be available until Wednesday. 
I'll call back then. 
We should study tonight then. 
Is the landlord going to install an air-conditioner?
Yes, she'll do it before we move in.
How do you like working in the London office?
 She works out at Speigel's Gym.
Weren't the pictures taken in color?
No, the basement is being renovated.
 I'm not quite finished looking at the menu. 
Sure, when do you think it'll arrive?
Who'll manage the Pearson project? 
We'll see once we get there.
Jasmine will oversee it for now.
We'll see once we get there.

What is the Trumper Corporation? In downtown Beijing. We'll meet with them today. It's a manufacturing company. Hello, I'd like to get an application for employment. Yes, he works in advertising. From March 1st to February 28th. O.K., just go to customer service. Didn't they cancel the football game? Alright, we can go then. No, it was just postponed. Please speak with a counselor.
Excite Travel is offering a four-day cruise to Puerto Rico and the Virgin Islands.



連読
the London office
Her office is
An aisle seat
she'll do
 move in.
Do you think you could 。。。
How much is it?




When are the sales forecasts due?
I have no idea what movies are featuring now.
The subway line two goes straight there.
Which restaurant should we eat lunch at?
That French cuisine was great.
Either one is fine.
Don't we need to leave for the airport soon?
Yes, right after I make one call.
I believe it begins at 11:00.
I won't change my mind.
Who is going to take charge when Stacy leaves?
I have to get permission first.
I was about to. Would you like some?
It is hot in here, isn't it?
No, I'll turn the heater off.
Aaron's last day of work is Thursday, isn't it?
No, the part hasn't arrived yet.
Shouldn't you dial nine to reach the hotel operator?
No, it says to press zero.
Wasn't the side table more useful in that corner?

Has team2's new campaign been researched? 
Yes, sometimes cooperation means everything. 
Their campaigns are quite different.

Both of them don't really attract me. What else is on?
Yes, I'd love to.
Would you like to watch the sitcom or the news?
Who explained this incident to you?

I didn't come that early. I was on time.
So he can get overseas calls.
How shall we celebrate Mr. Kim's promotion? 
Why don't we throw him a party
Since you two worked on the last project, this should be easy. 
I met both of them last year. 

Do you need to leave now or can you stay until Anna comes? 
No, you don't have to. 

Wasn't the furniture supposed to be here today?
I thought so.

Aren't our pictures supposed to be at the top of the page? 
I still have five more pages to go through. 
We changed it and put them at the back. 
You look great in that picture.


Do you know if the entire staff needs to submit a time sheet?
 I thought you haven't finished yet.

The restuarant was full and i couldn't find out to reservation.
a. where did you have dinner.
b. so when did you leave.
c. i have a reservation for trip




in recent years.
a very special guest host
over the last century.
Your mentor will be there to 
on a number of factors
to the staff room and your lockers.
and I am here to present the first-year nurse's orientation
All Master Thigh Products are made from the best quality materials and components, and are put together by trained workers. 
 Never before have so many of the greats of modern jazz come together under one roof to entertain, dazzle and delight. 																
I have no doubt 
time for Riley radio's hourly weather report.
don't forget to take your umbrella with you.
But just when you thought it could not get better, it does.































Congratulations on buying the Master Thigh 3000. Please note that all Master Thigh products come with a full one year guarantee. Before using this product, please understand the following in full. The Master Thigh Corporation takes no responsibility for any injury that may be suffered or sustained while using the said product. All Master Thigh Products are made from the best quality materials and components, and are put together by trained workers. In the unlikely event that something should go wrong with your Master Thigh product in the first year after purchase, please call the following number: 18005855858 and follow the prompts, or visit our website www.masterthigh.com and follow the instructions. We trust our product will give you the firmly shaped thighs you deserve.

Welcome to the Saint Mary's Hospital. My name is Senior Nurse Betty Nason, and I am here to present the first-year nurse's orientation. As fresh graduate nurses, you will all be assigned a mentor from the senior staff. Your mentor will be there to guide you through your first few months in the hospital. You will also be subject to monthly evaluations conducted by me and your respective mentors. You will be evaluated on a number of factors, including bedside manners, understanding the patients' needs, punctuality, professionalism and your ability to conform to the rules of the hospital. Your pay raises will be based on successful evaluations. Now follow me as I show you to the staff room and your lockers.												

Tonight I'm very proud to tell you that we will be bringing the very talented songwriter, Missy Waters. She has just returned directly from a very successful tour of Europe and I have no doubt Missy will dazzle you with skills as a jazz singer and songwriter. But just when you thought it could not get better, it does. Missy's brother and renowned jazz trumpeter, Jackie Waters, will also be joining Missy on stage making it the first time the pair has played together in 25 years. On drums we will have Charlie Marker, and on the bass, we will have Boom Boom Jones. Never before have so many of the greats of modern jazz come together under one roof to entertain, dazzle and delight. So, without further ado, let's give a very special welcome to the four.

It's now 6 o'clock, time for Riley radio's hourly weather report. If you're just about to leave your house, don't forget to take your umbrella with you. The drizzle is expected to start throughout the morning, so prepare to protect yourself. However, the sky will clear this afternoon, and the temperature will be just 18 degrees Celsius. It will be a much cooler day tomorrow with a high of only seven degrees, the coldest day in several months, so you'll definitely need a warm coat when you head out in the morning. That's the morning weather report on Riley radio. Now, back to our regular pop music program.



On today's program, we will be talking to Cheri Gardner, a well-known film critic in Hollywood. She will be discussing changes in the film industry over the last century.
In particular, she will talk about why smaller independent productions have become much more popular in recent years. And at noon we will open the phone line so our listeners can give their comments on the subject. And before I forget, I have an announcement to make. I will be covering the elections in San Francisco tomorrow so we will have a very special guest host, my own daughter.


Good morning, commuters. This is YQN104 traffic report. On Highway seven going south, traffic is moving slowly due to fog which is still heavy in some places. If you're heading south, we recommend taking Highway 17 as an alternative route since foggy conditions have already cleared on this road. Remember that highway construction is scheduled to begin tomorrow along Highway 17 from Cherry Street up to the Delta Road exit. You may want to avoid that area for the next few weeks.
Now, after our commercial break we will be with Robin for the sports updates. So please stay tuned.




地名
Puerto Rico　波多黎各
Virgin Islands　维尔京群岛
Des Moines, Iowa　爱荷华州　得梅因
Merchant City 
Trenton 特伦顿

**Comment by rainit2006** [comment]
I tend to think attending conferences is a good idea. You never know who you'll meet at one. For instance, just last week I gave a presentation at the Operations Conference. The presentation was on operations efficiency – ways in which businesses can streamline their processes and be more organized and efficient. Anyway, afterwards, a man came up to me and he was so excited by my presentation that he wanted to talk to me about writing a book with him. He also works on operations efficiency. We're actually going to meet tomorrow to talk seriously about the book. So, my point is just that you really never know who you'll meet at a conference, and what kinds of opportunities might arise as a result of a chance meeting.
-------------------------------------------------------
-------------------------------------------------------
Attention passengers for Skyway Airlines flight 610 to Portland: there's been a gate change. Flight 610 is now scheduled to depart from gate B-three. Due to a mechanical problem with an earlier flight, Skyway Airlines flight 610 to Portland will now be departing from gate B-three. We apologize for the inconvenience. The scheduled departure time for flight 610 remains 2:35. Boarding for flight 610 will begin at approximately 2:05. Again, Skyway Airlines flight 610 to Portland will depart from gate B-three at 2:35; boarding will begin at approximately 2:05.
-------------------------------------------------------
-------------------------------------------------------
We increased the budget for the coming year because we recognize that the company needs the money if it's going to maintain the momentum it achieved this year. I want you to know that the Board of Directors also recognizes the many important projects that were started during the past year, as well as the many exciting changes in the company's direction and focus. We hope the increased budget is a strong signal that we support these projects, and all the new changes, and believe in the potential of the company. We know you have a lot of work to do over the next three months - particularly completing the strategic planning document. The strategic planning document, as you all know, is really the blueprint for the company, and will lay out the way forward for the next few years. We are very anxious to see what you come up with.
-------------------------------------------------------
-------------------------------------------------------
You really need to do your homework before opening a new business. My suggestion is to get your hands on as much basic data as you can about the community where you plan to set up shop. Good sources of data include federal, state, and local governments. Governments need to know a lot about the populations they serve, so they periodically collect data through surveys and census programs. They collect data about all sorts of things, including age, family size household income, education levels – all sorts of things. And the data is available to the public through various government offices. Now, using all this information, you can put together a picture of the demographics of your potential customers. This can be very valuable in helping you to determine whether your business is likely to succeed in a particular area.
-------------------------------------------------------
-------------------------------------------------------
Hi, Susan, this is Lisa. I just wanted to make sure you had directions to my office in Clark Towers. There are two buildings, Tower One and Tower Two. The tower complex is next door to the art museum. My office is on the sixth floor of Tower Two. So, if you park in the South Street garage you'll enter Tower Two through the main entrance. There are signs in the parking garage, so it's pretty easy. Just go in the main entrance, walk to the end of the hallway, and take the elevator to the sixth floor. I'm in the last office on the right at the end of the hallway. It's suite six-ten and the door is marked Landower Group. Just walk in and ask the receptionist to page me. Okay, I'll see you at five o'clock, and we can go to dinner from there. I'm looking forward to it.
-------------------------------------------------------
-------------------------------------------------------
Making changes to an organization's filing system is a big job, and not one to be taken lightly. The best thing we did when we updated our filing system was to gain the support of both the administration and the people who use the files most often. Administrative support added legitimacy to the project, which turned out to be a key component to the success of the project. Also, getting the support of the end users really made the transition a lot easier. The people who use the files most were able to tell us which aspects of the old system worked well, and which needed to be changed. You really need the input of those who will be affected.
-------------------------------------------------------
-------------------------------------------------------
I want to burn theses lectures onto one CD, but I'm having trouble doing it. I keep getting an error message.

Have you checked all the settings on your software program?

I did, but it's the first time I've used this software and maybe I missed something.

Let me take a look. Okay, here's the problem. You had the wrong box checked for the file-type. You need to mark the box that says “MP-three". Try copying again, it should work this time.
-------------------------------------------------------
-------------------------------------------------------
Good morning, I'd like to reserve 25 rooms for a conference that I'm organizing in June.

Okay, I can help you with that. Do you want to reserve a room for a reception on the night that the attendees arrive?

No, we're holding the conference at Carlson College. That's all taken care of. We just need to find a place for the attendees to stay. The guests will be checking in on the Thursday the sixth and leaving on Saturday the eighth.

Very good. With that many rooms, I'm sure we can offer you a good deal. If you give me your e-mail address, I'll put together a price quote and send that off to your later on today.
-------------------------------------------------------
-------------------------------------------------------
Have you given much thought about saving for your retirement?

Well, I spoke to a financial planner last year. She helped me set up a retirement account and now I have money automatically deposited into it each month. It's pretty easy.

I'm just now talking to a financial planner. I think I'll wind up doing something similar to what you've done.

You really should. Like I said, it's easy, and it's automatic. The money is deducted right from your paycheck. It's a little shocking the first time you get your paycheck, and it's less than you normally get, but you get used to it fairly quickly. And the other side of it is that the retirement fund grows quickly too.



give much thought to
～について熟慮する［よく考える］、～を［について］よく［十分に］考慮する、～をおもんぱかる
-------------------------------------------------------
-------------------------------------------------------
Next Friday's your last day of work here, isn't it?

Yes, and I start my new job the following Wednesday.

That's not much time between jobs. Why don't you take some time off before you start your new job? You must have some unused vacation time.

I was going to, but I also promised John, my boss, that I'd finish the Longmont project before I left. I've been working on it since it started last year, and it's almost done. I'd like to see it through to the end.
-------------------------------------------------------
-------------------------------------------------------
Hello, I’m calling because I need to rent a table saw. Do you do that sort of thing?

Yes, we do. Do you want me to reserve one for you? It’s $35 per day, with a $200 deposit. You’ll need a major credit card and a photo ID when you come pick it up. And the saw’s due back no later than 24 hours after you pick it up.

Okay, that sounds good. I’d like to reserve one for one day. I’ll be in to pick it up at 5 p.m.

Okay, now if you’ll just give me a minute, I’ll pull up the form on the computer.
-------------------------------------------------------
-------------------------------------------------------
I’m interested in your packaging system, but it’s so much more expensive than your competitors’.

You’re right. Our prices are definitely higher than systems from other brands in the same category, but every system we sell is supported with extensive on-site training, and the best long-term service plan in the industry. That’s all included in the price I just gave you.

What makes the service plan so superior?

We’ll send a technician out to your site every six months to calibrate the equipment and do routine maintenance. This alone should eliminate your need to make service calls. But if you ever do need service, we’ll send a technician out the same day, at no charge, for as long as you own the system.
-------------------------------------------------------
-------------------------------------------------------
How’s your business doing?

Well, sales are definitely lower than expected. I think that’s because the housing market has been slowing down recently. In the appliance world, even small changes in the housing market tend to affect us a lot.

Hmm… I guess I never made the connection between the home appliance industry and the housing market before. But it makes sense.

It sure does. When people buy new houses they usually buy new appliances for them. In fact, you can accurately gauge how the housing market is doing just by looking at our sales figures.
-------------------------------------------------------
-------------------------------------------------------
The copy machine isn’t working again. Has anyone called in about it yet?

Yes, I spoke to the repair man this morning. I told him the paper’s still getting stuck in the same spot as last week when he was here. He said he’d be here around 11.

Oh good, you were able to talk to the same man. He seemed very competent. I’m surprised it’s broken again.

It’s not his work that’s the problem. Last week he told me we may need to replace a part. If that’s true, it might make more sense to just replace the whole machine. This one’s at least five years old. New ones aren’t that expensive.
-------------------------------------------------------
-------------------------------------------------------
My name is Steve Sherman. I reserved the meeting room from 8:30 until 5 o’clock today. I was told to come to the counter to get the key.

Hi Steve, I’m Diana. We spoke on the phone when you called yesterday. It’s nice to meet you in person. Hold on. Here’s the key to the room.

Thanks. Now, I also ordered the catering package – coffee and sandwiches.

Yes, that’s all taken care of. They’re setting up the first coffee service now, and the sandwiches will be available at noon. There’ll be a second coffee service at 2.
-------------------------------------------------------
-------------------------------------------------------

-------------------------------------------------------
-------------------------------------------------------

-------------------------------------------------------
-------------------------------------------------------
Where is everybody today? It's really quiet in here.

Fridays in July and August tend to be like this. Lots of people taking vacation days or half-days to get out of the heat of the city. On top of that (除此之外), there's some kind of promotional event being hosted by one of our vendors in the cafeteria.

Right. Marge and Henry just told me that they were on their way over there. But still, it's almost too quiet. In a way(在某种程度上，在某一方面) it helps you concentrate on your work, but at the same time, it's so quiet it feels odd, like being here on a Saturday. And that doesn't help me concentrate.
I know what you mean. Why don't we stop by the cafeteria ourselves to see what's going on?
-----------------------------------------------
-----------------------------------------------

Hi,Charlene. Aren't you going to the team meeting today? No. Simon just sent an e-mail saying it's not being held today. There's a crisis in the tech department and they can't afford to give up the time. What's the crisis? Is there anything I can do to help out? I'm afraid not. It's a programming issue. The client just realized that they sent us the wrong data for the computerized reports they want. So now we have to re-do three weeks of coding in three days. What does the man learn? How can the man's attitude be described? What problem is mentioned?
-----------------------------------------------
-----------------------------------------------
There's no relief in sight from the hot and dry conditions. Today's high will be in the mid-nineties – the eighth straight day of plus-ninety temperatures. And tomorrow, Saturday, will be worse – expect temperatures to hit 100. And it doesn't look like there's any hope of a thunderstorm until perhaps Wednesday or Thursday. But as we all know from looking at the brown grass on every one's lawn, we need more than a few scattered thunderstorms to help us out, and to get us out of the high alert for brush fires that we are currently in. Later in our broadcast, we'll discuss some of the new water restrictions that will be going into effect on Monday on the North Side and on Tuesday on the South Side. So stay tuned to find out how these restrictions will affect your neighborhood and your lawn. How will the next day's weather compare to the current day's? When is rain predicted to arrive? What will be discussed later?
-----------------------------------------------
-----------------------------------------------
Okay, here's your key. The elevator is down the hall to your right. Breakfast will be served from seven to 11 am in the restaurant, which is down the stairs. The fitness room is from eight am to ten pm. There is a pool and sauna. They also are open from eight to ten. You need to bring a towel from your room, as none are available at the fitness room. Thank you for choosing the Golden Hotel, and I hope you have a pleasant stay.
-----------------------------------------------
-----------------------------------------------
Hi, Sally, this is Bill Sullivan. I'm calling to find out if you sent out the blueprints for the O'Malley Arena yet. We're sort of waiting for them. Hi, Bill. I'm sorry, I haven't sent them yet. I got caught up in other work. I'm really so sorry. That's fine, but I really need to put a little pressure on you. We're at a place right now where we can't move forward without the blueprints. Could you please make this a priority? Yes, Bill, I'm so sorry. Tell you what, I'm going to drop what I'm doing right now and make sure the blueprints are ready to go. They'll go out by overnight courier this afternoon. I promise you. 
-------------------------------------------------
-------------------------------------------------

Thank you for staying tuned to Talk Tonight. Our special guest this evening is a graduate of West Point. He was a former intelligence officer and retired lieutenant general who served under two presidents. This man is also an expert on Russia and frequently lectures at universities across the nation. He is here this evening promoting his new book, Behind the Kremlin, which gives us insight on Russian politics. It is currently on the best seller's list. It's my pleasure now to welcome Mr. Ivan Strand.
-----------------------------------------------
-----------------------------------------------
O.K., I'm going to hand out your training packet which will give you sample scripts and show you how to deal with specific situations. The most important part about your job is to be polite, speak clearly, and assure the people you're calling that you're calling to perform a survey, not to sell them something. Most people get upset when they speak with telemarketers. However, they are more likely to stay on the line if you explain that you were taking a survey. Now, before we begin, I want you all to listen to this call. Take notes because I want to know how you would have handled the situation. 
How does the woman start her speech? 
What is important about the job? 
What is the audience instructed to do?
-----------------------------------------------
-----------------------------------------------
The only way we can truly compete with our top competitors is by finding a way to accept the deal. I know that many of you feel that Creation Inc.'s suggested bid of $20 billion undervalues the company, and is in fact, much lower than you expected. However, it is important we find common ground. Frankly, we're not doing so well, as you can see in our share prices. This deal will not only help them, it will help us immensely. Our shares are continually slipping while others are gaining. We should keep the investors in mind when negotiating with Creation Inc. and do our best to reach an agreement.
-----------------------------------------------
-----------------------------------------------
Clear View Optical is having a sale on all sunglasses. You can save up to 70% on our entire stock of sunglasses, including our new shipment of designer wear. If you haven't had your eyes checked, and aren't seeing clearly, we'll test your vision and help you find the right glasses. Our experts know the latest trends and will help you find the perfect fit for your face shape. For more information or to make an appointment, call 8343894. Don't let blurry vision set you back. Visit Clear View Optical and see sharp images again.

-----------------------------------------------
-----------------------------------------------
Attention everyone, if you would, please pay attention so we can form lines to bring a little more order to entering the museum. If you already have a ticket, whether it's a seasonal ticket or online ticket, please proceed to line one. If you are part of a tour group or would like to purchase group tickets, please proceed to line two. And last, if you don't have a ticket, please proceed to line three. Standard admission is $12 per adult and $8 for children under 16. Student and senior citizens can also get a special rate of $10. The museum will open in ten minutes. Thank you for your patience and cooperation in making entrance to the museum run smoothly.
-----------------------------------------------
-----------------------------------------------
The Krugman R-99 printer is your best bet since you're looking for a new printer for your home office. Not only does the R-99 allow you to print, fax and make copies, but it's also compact, so it doesn't take up much space. Its retail price is $219.99, but it's on sale this week for $159.99. I just bought one for my son who's going away to college this fall. The R-99 also comes with a two-year warranty, which is longer than most printer warranties. If you like, I can show you some print samples. What is mentioned about the printer? What is the printer's sale price? What does the speaker offer to do?
-----------------------------------------------
-----------------------------------------------
This fall, make sure you fasten your seatbelts, because it's going to be a bumpy ride. Peter and Tasha are back with an even tougher race than before, and this time, more than a-million-dollar cash prize is at stake. Get ready for the big day. On October 4th, the speed will be high, the crowd will be cheering, and the blood will be flowing. Richard Burbank and Ivana Stein are back and ready to race in another high-speed adventure that is sure to keep you on the edge of your seats in Velocity Two, this fall, at a theater near you.
-----------------------------------------------
-----------------------------------------------
This press conference was called to clarify some talk that has been going around. Ocean Blue Air and Sky High Airlines are in talks to discuss an alliance not a merger as some newspapers and stations have reported. Although our board members are meeting this week, we're not expecting to have any decisions made anytime soon. We just want to see what options we have. With fuel prices breaking records and lower profit expectations, Ocean Blue Air must do its best to stay on top, which means examining what options are available. Now, I will be happy to answer a few questions.
-----------------------------------------------
-----------------------------------------------
Min-soo, I'll be going out of town for business next week. I was wondering if you could watch my cat for the seven days that I'm gone. Your cat? I didn't know you had one. Sorry, but I'm actually allergic to cats. Yeah, he stays inside most of the time. Well, do you know anyone who could watch him? I asked my brother, and he can't do it either. Why don't you put him in a kennel or boarding house? I don't think it would be too expensive to put your cat there for a week. But first, I'll try calling my little sister. I think she likes cats. How long will the woman be gone? Why does the man decline to do the favor? What does the man offer to do?
----------------------------------------
Did you see the weather for this weekend? We're supposed to have stormy weather starting Friday morning. Should we postpone our department outing? We could move it to the following Saturday. I wouldn't be able to make it because I'm going out of town that weekend. Why don't we just cancel the afternoon activities and keep it to lunch? That's a good idea. Do you think the others will mind? I think they would welcome the idea. We've had so much work this week. Everyone is exhausted. I'll go talk to everyone and let them know what's going on. Could you also call the restaurant and make sure that we're seated inside? 
-----------------------------------------------
-----------------------------------------------
Hi Tom, I tried installing my new car stereo, but I guess I'm just not good with electronics. Do you think you can help me? I've never installed a car stereo before. I'm not sure I'd be able to do it. Oh. I figured since you were able to wire your home system, you could install a car stereo. Do you have any friends who can do it? I'm afraid a professional might be expensive. Well, let me look at the directions first. I'll stop by later this afternoon and try to do it, and if I can't, you should have it done professionally. I don't think it's very expensive at all. What is the purpose of the call? What are the speakers discussing? What does the man agree to do?
-----------------------------------------------
-----------------------------------------------
I agree. But there's been so much going on over the last month, there's always been a schedule conflict for someone who is crucial to the meeting. Once we get over the busy period I'll set up a fixed recurring meeting schedule.
-----------------------------------------------
-----------------------------------------------
Have you seen Rachel today?

Not yet. But it's only 8:45. I'll let her know you're looking for her when she gets in, which should be any minute now. Is there anything I should tell her?

Yes. We have an urgent situation with one of the pieces of art that was supposed to be revised for the Henderson poster. It should have been in yesterday afternoon. When I didn't see it this morning, I called the art vendor and they said they never received the revision request. This poster is supposed to go back to Henderson for confirmation today.

I'll let her know. But try not to worry. If the revision is minor we should be able to do it in-house first thing this morning.
-----------------------------------------------
-----------------------------------------------
What a fantastic rug, Karen. Did you buy it while you were traveling in India with your parents last year?

No, I made it myself.

You made it yourself? That's incredible. How did you find the time to do something like that?

Well, you know what they say – you make time for things you enjoy. I took a rug-making class a couple of years ago, but it wasn't until I was in India and went to a rug market that I decided to finally make my own. So you're right. India is connected to this rug. But I didn't buy it there, just the materials I needed to make it.
-----------------------------------------------
-----------------------------------------------
Where is everybody today? It's really quiet in here.

Fridays in July and August tend to be like this. Lots of people taking vacation days or half-days to get out of the heat of the city. On top of that, there's some kind of promotional event being hosted by one of our vendors in the cafeteria.

Right. Marge and Henry just told me that they were on their way over there. But still, it's almost too quiet. In a way it helps you concentrate on your work, but at the same time, it's so quiet it feels odd, like being here on a Saturday. And that doesn't help me concentrate.

I know what you mean. Why don't we stop by the cafeteria ourselves to see what's going on?
-----------------------------------------------
-----------------------------------------------
Hi,Charlene. Aren't you going to the team meeting today?

No. Simon just sent an e-mail saying it's not being held today. There's a crisis in the tech department and they can't afford to give up the time.

What's the crisis? Is there anything I can do to help out?

I'm afraid not. It's a programming issue. The client just realized that they sent us the wrong data for the computerized reports they want. So now we have to re-do three weeks of coding in three days.
-----------------------------------------------
-----------------------------------------------
There's no relief in sight from the hot and dry conditions. Today's high will be in the mid-nineties – the eighth straight day of plus-ninety temperatures. And tomorrow, Saturday, will be worse – expect temperatures to hit 100. And it doesn't look like there's any hope of a thunderstorm until perhaps Wednesday or Thursday. But as we all know from looking at the brown grass on every one's lawn, we need more than a few scattered thunderstorms to help us out, and to get us out of the high alert for brush fires that we are currently in. Later in our broadcast, we'll discuss some of the new water restrictions that will be going into effect on Monday on the North Side and on Tuesday on the South Side. So stay tuned to find out how these restrictions will affect your neighborhood and your lawn.

How will the next day's weather compare to the current day's?

キーワード：relief in sight; Today's high; 
 xxxx straight day: 連続xxx日間
----------------------------------------------------
----------------------------------------------------
Charleston Light and Gas has announced plans to raise its rates in time for the winter season. The plan, pending regulatory approval, would raise gas prices by approximately seven dollars a month for most residential customers. Charleston officials said the rate hike is necessary to keep up with the increasing prices of oil and natural gas. The State Resources Board, the public agency responsible for regulating utilities, plans to discuss Charleston's request early next week. Should the board approve the rate hike, it would go into effect on the first of December. By about how much will monthly prices rise? What reason is given for the rate hike? When would the proposed rate hike take place?

rate hike: 参考：　interest rate hikeの意味は利率引上げ
keep up with...
go into effect (on the first of December), 生效，开始实施

Should the board approve the rate hike,...
仮定法の文のIf節（仮定節）はIfを省略して助動詞を倒置しても意味は同じす。
If I were a bird , I could fly to you .（過去）
→ Were I a bird , I could fly to you .
（　もし今私が鳥ならばー　）
If I had been rich , I could have bought the house .（過去完了）
→ Had I been rich ,I could have bought the house .
（　もしあのとき私が金持ちだったらー　）
It it should rain tomorrow , I'll stay home.（未来）
→ Should it rain tomorrow ,I'll stay home.
（　もし万一明日雨が降ればー　）
質問の文は仮定法未来の文の倒置簡易形ですね。
if your main e-mail server should go offline for any reason ,
→ should your main e-mail server go offline for any reason ,
（　もし万一何かの理由であなたのメインメールサーバーがオフラインになったらー　）

------------------------------------------------------------------
------------------------------------------------------------------
As part of our general IT upgrade, we've been seriously overhauling our security infrastructure, partitioning information across multiple servers, using new encryption schemes and so on. So one of the ways this is going to impact all of you is, well, aside from making your data more secure, we're adopting a new password policy, and we're asking everyone to please understand it and comply with it. Basically, what the policy means is every password you use within the corporate network – whether it's for your employment records or project files – all passwords now have a lifespan of one year. That means that once a year, you're going to have to change all your passwords. I know it's a small inconvenience to you, but it adds a very important layer of network security and I hope you'll all agree with how important that is.


as part of xxxxx, 作为一部分
overhaul ， 〈組織・方法・考えなどを〉徹底的に見直す.
partition  n 分割，分区  v，〈…を〉〔…に〕分割する，区画する 〔into〕.
comply with it, 承諾；従う；


---------------------------------------------------------------
---------------------------------------------------------------
As a personal fitness trainer, I have to tell you that one of the worst things you can do when you're working out at the gym is drink soda. Aside from making you feel bloated, the sugars in the soda keep your body from absorbing water, which can make your muscles cramp up. I mean, I don't even recommend those so-called juice drinks that contain a bunch of added sweeteners. Stick to the natural stuff. 100% juice is great. It contains fructose, which helps you rebuild your muscles and vitamins and like that. If you want, you can make your own sports drink by diluting some juice and adding a little salt. I add about a teaspoon for every quart of diluted juice. This helps keep you hydrated, and helps you replenish the salts you sweat out. 

bloated, ふくれた，太り過ぎた，むくんだ.
absorb, 〈水分・熱・光などを〉吸収する，吸い上げる[込む].
cramp, 使痉挛，使抽筋； 
sweetener,  (人工)甘味料.
fructose, 【化学】 果糖.
diluted , 希釈する, (濃度が);薄い, 希薄な
hydrate,  n 水和物，水化物. v 〈…を〉水和させる，水酸化させる.
replenish, 〈…を〉補充[補給]する
stick to 
Aside from は別として
a bunch of　一堆
keep from doing〈人・物が〉〔…するのを〕妨げる，させないでおく
------------------------------------------------------------
------------------------------------------------------------
That's so annoying. I just bought this expensive printer, but it doesn't come with the USB cable I need to connect it to my computer. Now I have to go back to the computer store just to get the cable. I know, why do they do that? They should just include the cable with the printer. That's what I think. I can't believe I have to make an extra trip just to get a cable. Well, if you're going back to the store anyway, you might want to buy some ink cartridges, too. The ones that come with the printer are so-called starter cartridges. They're not regular full-size cartridges, and they'll run out after you print a few dozen pages.


cartridge, 打印机墨盒；(銃の)弾薬筒，装弾 《実包》; 
-----------------------------------------------------------
-----------------------------------------------------------
Our restaurant's been open about two years, and we were just voted one of the best restaurants in the city. We're riding high, so it has me wondering if maybe I should use this as an opportunity to expand my business. What do you think? Well, first of all, congratulations on your reward. That's no small achievement. Now as for expansion, the main thing you need to think about is what that'll do for you and your staff. I mean, how much do you oversee operations right now? Exactly, that's my main concern. I'm in the restaurant every day, and I oversee practically every detail. I definitely couldn't do that if I opened another restaurant. No, you couldn't, which means you'd have to hire or train somebody you could trust. The main question is, what's the ideal size you can handle, and still maintain your quality.
--------------------------------------------------------------
---------------------------------------------------------------
I was just trying to pay my parking fees, and the machine I was using ate my money. It's the machine on the second floor parking deck. Huh, that's strange. It's never done that before. Do you have your parking ticket with you? No, the machine ate that too. I put in the ticket, the machine told me I owed two dollars, I put in a ten-dollar bill, and the machine crashed completely. It won't respond to any button I press. Okay, that's definitely weird. Let me close down this booth and I'll go upstairs with you to check it out. Hold on a second. I need to lock up here. 
--------------------------------------------------------------
--------------------------------------------------------------
I love the office buildings around here. You see that one across the street with the nice ledge? Look at those huge windows. Wouldn't that be a great place for an office? You know, there are a lot of vacancies in these buildings. If you're serious about getting an office around here, I think the real estate values are actually pretty good. The prices are low now. Really? Even for a historic building like that? I mean, look at that thing. Well, remember, this city's economy has been depressed for a long time. It's definitely turning around slowly, but I don't think there's a lot of confidence in the market yet. 
--------------------------------------------------------------
--------------------------------------------------------------
Wow, this is really strange. I just had a resume cross my desk, and the name on it is a guy who used to be my boss at Booker. Isn't that bizarre? This person used to be your boss, and now you're looking over his resume? That is pretty weird. Which job is this for? It's for a manager of client services position with the supply group. Actually, I think he'd do a pretty good job. I really liked him as a boss. I wonder what happened to him at Booker. Well, if you think he'd be good, there's nothing wrong with bringing him in for an interview. It'll probably surprise him to hear from you. 
----------------------------------------------------------
----------------------------------------------------------
At Dresher First Bank, we know that small businesses make our community run. That's because we've been dealing with small businesses for more than 120 years. Dresher First Bank offers a number of products tailored to the needs of small business owners. Like our standard checking account, which features no minimum balance, no monthly fees, and lets you write up to 300 checks a month. At Dresher First Bank, small business owners benefit from our years of commercial lending experience, all in the local community. Combine this with our outstanding array of competitively priced loans, and you'll see how Dresher First Bank helps the small and independent businesses that make our community run. Come by and see us sometime. We're here to help you.


tailored to the needs of: 

---------------------------------------------------------
---------------------------------------------------------
You've been named one of the top architectural design firms in the Midwest. Tell me, how long has your company been in business?

From 1991 until 2001, we, I should say, I operated as a sole proprietorship. I hired subcontractors as needed, but did most of the work myself. I formed a corporation in 2002, due to all the design work I was getting.

How many employees do you currently have?

We work with a group of regular subcontractors, architects, interior designers, contractors and so on, who get together on a project-by-project basis. We find this flexibility allows us to address the specific demands of each project, regardless of its complexity or scope.

In which field does the man work?

What does the man imply about his business?

What does the man say is a benefit of his business model?
--------------------------------------------------------------
--------------------------------------------------------------
I'm afraid there's no way we can allow you to use any part of any of our client projects in your portfolio, I'm sorry. Having said that, I do recognize that having samples to provide as part of a portfolio is important. So if you can do it while protecting our client's confidentiality, I'm willing to be a little flexible. One thing I've suggested to others in the past is to mock up something that is similar to the client's materials, but that strips out proprietary content and logos. In addition to stripping out proprietary content and logos, the other thing you need to do is to not distribute your portfolio electronically, for example, on web sites, or through e-mail, etc. I would limit the portfolio to one-on-one distribution, preferably by regular mail, to industry folks who are interested in working with you.


portfolio, 公文包； 代表作品集； 证券投资组合； 部长[大臣]的职位. 网络. 投资组合；; 历程档案
mock up  実物大の模型 〔of〕.
strip out はぐ，取り除く.
proprietary, 所有主の，所有の;独占の，専有の; 専売の.
preferably, 好んで，むしろ，なるべく(なら).I want to get married; preferably to a rich man. 私は結婚したい, なるべくならお金持ちと.

--------------------------------------------
-------------------------------------------
Good afternoon. This is Craig Morano with a WBX Radio sports update. The Dazzlers beat the Bearcats 106 to 74 in last night’s basketball game at Renfrew Stadium. Dazzler’s small forward Derrick Livingston scored 18 points, including a 3-point basket with only 15 seconds left in the game. Dazzler’s power forward and leading scorer, Justin Marlboro, who entered the game with an average of 21.3 points per game, finished against the Bearcats with20 points and eight rebounds. This is the first game the Dazzlers have won by such a large margin. The Dazzlers are scheduled to play the Memphis Snakes next Friday night at 7 p.m. at Tarrytown Stadium.


small forward  小前锋
power forward  大前锋
leading scorer 头号得分手
rebound, 篮板球
-----------------------------------------------------
------------------------------------------------------
Antram Incorporated is instituting a new recycling program. Our first step is to place sets of recycling bins around our facilities. A set is made up of three separate recycling bins that will be dedicated to the collection of aluminum cans, plastic bottles, and newspaper and other paper. There will be four sets altogether. One will be placed in the first floor operations break room, one will be placed in the second floor administration break room, and two will be placed in the cafeteria. Depending on the overall success of this program, additional recycling collection points may be added. Your participation in and support for the program is greatly appreciated.

-------------------------------------------------------
----------------------------------------------------------
I want to discuss the changes in assignments and reporting that will go into effect October 1st. First of all, your team won't be reporting to Lisa any more. Rebecca will be your new supervisor. Lisa will still be working with your team, but she's not going to be involved with the day-to-day details or schedules of any of your projects. Her focus is going to be on expanding our client base and creating more revenue. The second change is a shift of resources within the team. Let me give an overview and then we'll go into details. First, Carol and John are going to take over the Slavin Project. This will free up Steve to be a co-lead on the Talon Project with Marcia. Robert and Wendy, you'll both remain on your current projects in your current roles. I'm sure you all have a lot of questions. And I'll do my best to try and answer them.

-------------------------------------------------------
----------------------------------------------------------
The Lester Water Department is flushing the municipal water system this week, starting Monday, between the hours of midnight and 5 am, and lasting through Friday morning. Flushing is performed every spring as regular maintenance to remove mineral deposits and sedimentation that has collected over the winter months. Residents are asked to let the cold water run from the tap for a full minute to ensure that it is clear. The water may be temporarily discolored but it is safe to drink and to use for cooking. For more information, call 3575439. When is the flushing scheduled to be completed? What is learned about the flushing? What are residents advised to do?

-------------------------------------------------------
----------------------------------------------------------


-------------------------------------------------------
----------------------------------------------------------


-------------------------------------------------------
----------------------------------------------------------
